src/core/scope.c

```diff
@@ -353,36 +360,78 @@ static void scope_enter_signal(Scope *s, ScopeState state, ScopeResult f) {
         scope_enter_dead(s, SCOPE_FAILURE_RESOURCES);
 }
 
-static int scope_start(Unit *u) {
-        Scope *s = SCOPE(u);
+static int scope_enter_chown_cgroup(Scope *s) {
+        Unit *u = UNIT(s);
+        pid_t pid;
         int r;
 
         assert(s);
 
-        if (unit_has_name(u, SPECIAL_INIT_SCOPE))
-                return -EPERM;
+        r = scope_arm_timer(s, usec_add(now(CLOCK_MONOTONIC), u->job_running_timeout));
```

Comment:
hmm, that's a surprising timeout to use. I think we probably should introduce a new field in scope units `timeout_start_usec` to match service units, and then use that for this purpose. that would be cleaner...

Reply:
So maybe we should use default manager's start timeout (`default_timeout_start_usec`)? Frankly, I don't think that another user configurable timeout is needed. I can't think of the case when the sys admin would want to configure such low-level detail, especially for operation that shouldn't take too much time anyway (chown() on cgroupfs should be very quick).

In any case, if you insist I will add the explicit option.

Reply:
looks good